.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: Lint
+
+on:
+    # Trigger the workflow on push or pull request,
+    # but only for some branches
+    push:
+        branches:
+            - master
+            - plugin
+    pull_request:
+        branches:
+            - master
+            - plugin
+
+jobs:
+    run-linters:
+        name: Run linters and TS checks
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Check out Git repository
+              uses: actions/checkout@v3
+
+            - name: Set up Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16
+
+            # ESLint and Prettier must be in `package.json`
+            - name: Install Node.js dependencies
+              run: npm i
+
+            - name: Pull starknet-hardhat-plugin
+              run: |
+                git clone https://github.com/0xSpaceShard/starknet-hardhat-plugin.git
+                cd starknet-hardhat-plugin
+                npm install
+                npm run build && npm link
```

Comment:
Why do just these two commands use `&&`? Why not have every command on its own line?

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-example/pull/122/commits/26251383faf1a638ab2e5af9da2f0f60417dcc78